.github/workflows/examples-nxp.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
               run: |
                   scripts/run_in_build_env.sh "\
                       ./scripts/build/build_examples.py \
-                      --target nxp-k32w0-freertos-lighting-factory \
+                      --target nxp-k32w0-freertos-lighting-factory-log-progress \
```

Comment:
What is the current flash increase from turning this on?  What steps are we taking to bring it down?

Ideally, we would bring it down _before_ enabling it on constrained platforms.

Reply:
started with #36172

In the end the extra code is to support interfaces that are not `index in a contiguous array allocated in ram`.

Reply:
#36229 is a followup, saves another 1K of flash.

Reply:
Resolving as I started a list of flash savings PRs in the summary of this PR.